ui/app/appointments/controllers/manage/appointmentsFilterController.js

```diff
@@ -7,6 +7,8 @@ angular.module('bahmni.appointments')
                 $scope.isSpecialityEnabled = appService.getAppDescriptor().getConfigValue('enableSpecialities');
                 $scope.isServiceTypeEnabled = appService.getAppDescriptor().getConfigValue('enableServiceTypes');
                 $scope.isFilterOpen = $state.params.isFilterOpen;
+                $scope.isCollapsed = appService.getAppDescriptor().getConfigValue('collapseServiceFilter') == true ?
```

Comment:
HI @nehagupta2123, Can we rename `isCollapsed` variable. The name suggests its a boolean field but seems like it's not a boolean field

Reply:
Also if you have time, can you create a method for `appService.getAppDescriptor().getConfigValue(<any string>)`